superset-frontend/src/components/Button/index.tsx

```diff
@@ -26,7 +26,7 @@ import {
 
 import { mix } from 'polished';
 import cx from 'classnames';
-import { AntdButton } from 'src/components';
+import { Button as AntdButton } from 'antd';
```

Comment:
I think our base components should be using the AntD component directly, not the re-exported version. In fact we shouldn't re-export these AntD components unless they're really needed. We should be using our Superset component if/when it exists, for consistency.